website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -24,17 +24,10 @@ brew install dbt-cloud-cli
 Verify the installation:
```

Comment:
I moved this up so you read it before executing commands. @greg-mckeon , is `.zshrc` and bash Mac OS specific? Should we make this more OS agnostic or add something Windows specific?
```suggestion
Verify the installation by requesting your homebrew installation path (not your dbt core installs). If the `which dbt` command returns nothing, then you should modify your PATH in `~.zshrc` or create an alias.
```

Reply:
this is mac or linux specific. there isn't really a more agnostic option, but most beta users are on mac. it seems ok to leave for now.